src/main/resources/templates/db_scripts/schema.json

```diff
@@ -3,21 +3,7 @@
     {
       "tableName": "template",
       "withMetadata": true,
-      "pkColumnName": "_id",
-      "generateId": false,
       "withAuditing": false,
```

Comment:
Can you remove the line `"withAuditing": false,`? This is redundand because it is the default.

Reply:
done